projects/nexus-dashboard/day-2/abhishekgoswami/README.md

```
@@ -0,0 +1,64 @@
+ ├── utils/
+ │    ├── formatters.js           # roundTemp, unixToTime, titleCase
+ │    └── localStorage.js
+ ├── styles/
+ │    └── index.css
+ └── App.jsx
+```
+
+### Setup
+1. Get a free API key at [openweathermap.org/api](https://openweathermap.org/api)
+2. Create `.env.local` in the project root:
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)